modules/tinymce/src/plugins/accordion/main/ts/core/Actions.ts

```diff
@@ -0,0 +1,52 @@
+import Editor from 'tinymce/core/api/Editor';
+
+import { fireInsertAccordionEvent } from '../api/Events';
+
+const nonRemovableContainers = [ 'BODY', 'TD', 'TH', 'LI', 'DIV', 'DT', 'DD' ];
+const isNonRemovableContainer = (node: Node): boolean =>
+  nonRemovableContainers.includes(node.nodeName);
+
+const validAncestorContainers =
```

Comment:
Is table really a correct parent. You shouldn't be able to insert an accordion next to a `tbody` ?

Reply:
Simplified the naming a bit and added a more explicit `getInsertPosition` function. Let me know if this clarifies this bit and please share your ideas about the naming!

Reply:
Obsolete due to the latest changes.